app/assets/javascripts/modules/external_users/claims/Dropzone.js

```diff
@@ -1,85 +1,200 @@
 moj.Modules.Dropzone = {
-  $target: {},
-  init : function() {
+  init: function () {
     var self = this;
 
-    this.$target = $('.dropzone');
-    this.$document_ids = $('.document-ids');
+    this.target = $('.dropzone');
 
     Dropzone.autoDiscover = false;
 
-    this.$target.dropzone({
+    if (self.dragAndDropSupported() && self.formDataSupported() && self.fileApiSupported()) {
+      this.target.addClass('dropzone-enhanced');
+      self.setupDropzone();
+      self.setupFileInput();
+      self.setupStatusBox();
```

Comment:
If `this` and `self` refer to the same thing, as per line 3, why use two different calls?